### PR TITLE
Bump submariner to 0.15.0-rc0

### PR DESCRIPTION
**Pull request by vthapar**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -32,7 +32,7 @@ require (
 	k8s.io/code-generator v0.26.3
 	k8s.io/component-base v0.26.3
 	k8s.io/klog/v2 v2.90.1
-	open-cluster-management.io/addon-framework v0.6.1
+	open-cluster-management.io/addon-framework v0.1.0
```
Comment (vthapar) [inline go.mod:35]: @skitt Any idea why this change? I used the `update-submariner.sh` script.

Comment (skitt) [reply go.mod:35]: No, I haven’t figured out why `go get` downgrades that dependency... You need to manually fix it.

**Comment by skitt** [comment]
This is failing because of the status update changes in Admiral. We can fix it either by populating the default scheme, or waiting for the upstream fix to be backported to the 0.15 branch and updating to that.

**Comment by tpantelis** [comment]
> This is failing because of the status update changes in Admiral. We can fix it either by populating the default scheme, or waiting for the upstream fix to be backported to the 0.15 branch and updating to that.

Just need another approval on https://github.com/submariner-io/admiral/pull/593.  @vthapar @sridhargaddam @aswinsuryan
